t/t2204-job-info.t

```diff
@@ -0,0 +1,109 @@
+#!/bin/sh
+
+test_description='Test flux job eventlog service'
+
+. `dirname $0`/kvs/kvs-helper.sh
+
+. $(dirname $0)/sharness.sh
+
+test_under_flux 4 job
+
+wait_lookups_nonzero() {
+        i=0
+        while (! flux module stats --parse lookups job-info > /dev/null 2>&1 \
+               || [ "$(flux module stats --parse lookups job-info 2> /dev/null)" = "0" ]) \
+              && [ $i -lt 50 ]
+        do
+                sleep 0.1
+                i=$((i + 1))
+        done
+        if [ "$i" -eq "50" ]
+        then
+            return 1
+        fi
+        return 0
+}
+
+#
+# job eventlog tests
+#
+
+test_expect_success 'flux job eventlog works (active)' '
+        jobid=$(flux jobspec --format json srun -N1 hostname | flux job submit) &&
```

Comment:
Suggestion: run `flux jobspec` once and save the output to a file for use in subsequent tests (see e.g. `t2202-job-manager.t`. This will slightly speed up the test.